typescript-local-storage/data.ts

```diff
@@ -0,0 +1,29 @@
+/* exported todos */
+interface Todo {
+  todoId: string;
+  task: string;
+  isCompleted: boolean;
+}
+
+let todos: Todo[] = [];
+
+function writeTodos(): Todo {
```

Comment:
You aren't returning anything from this function, so this is the incorrect return type.